app/drivers/sensor/cirque_trackpad/cirque_trackpad.c

```diff
@@ -0,0 +1,256 @@
+#define DT_DRV_COMPAT cirque_pinnacle
+
+#include <init.h>
+#include <drivers/sensor.h>
+#include <zmk/sensors.h>
```

Comment:
Hey, I'm curious, how do you compile it when `zmk` includes are not included in the CMakeLists?

Reply:
Can you clarify your problem? I'm having trouble understanding

Reply:
When I'm trying to compile with rebased main the compiler fails to find `zmk/sensors.h` include. And I was wondering if it's the case for you.

Reply:
Absolutely not. I'm guessing you somehow have a ZMK version that's way too old, or maybe the CMake stuff got messed up somehow?